### DISCUSSION TITLE
joining two distinct json data points

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#541] [Q&A]
Greetings,

let us I have a US Counties geojson file and another file with other details missing in the geojson file. Does this plugin allow for some sort of merge or join on a key data point shared by both?

Thanks

**Comment by yesoreyeram** [comment]
Hi @yosiasz. No it doesn't yet do joins. Can you submit a feature request with details of what you are trying to achieve with sample data?

**Comment by yosiasz** [comment]
Hello @yesoreyeram 

Here you go. Please let me know if you have any questions in that discussion?
https://github.com/yesoreyeram/grafana-infinity-datasource/discussions/542
